WooCommerce/Classes/ViewRelated/Orders/Order Details/Review Order/ReviewOrderViewModel.swift

```diff
@@ -33,15 +71,117 @@ final class ReviewOrderViewModel {
 // MARK: - Data source for review order controller
 //
 extension ReviewOrderViewModel {
-    var screenTitle: String {
-        Localization.screenTitle
+
+    /// Sections for order table view
+    ///
+    var sections: [Section] {
+        // TODO: add other sections: Customer Info and Tracking
+        return [productSection].filter { !$0.rows.isEmpty }
+    }
+
+    /// Filter product for an order item
+    ///
+    func filterProduct(for item: OrderItem) -> Product? {
+        products.first(where: { $0.productID == item.productID })
+    }
+
+    /// Filter addons for an order item
+    ///
+    func filterAddons(for item: OrderItem) -> [OrderItemAttribute] {
+        let product = filterProduct(for: item)
+        guard let product = product, showAddOns else {
+            return []
+        }
+        return AddOnCrossreferenceUseCase(orderItemAttributes: item.attributes, product: product, addOnGroups: addOnGroups).addOnsAttributes()
+    }
+
+    /// Product Details cell view model for an order item
+    ///
+    func productDetailsCellViewModel(for item: OrderItem) -> ProductDetailsCellViewModel {
+        let product = filterProduct(for: item)
+        let addOns = filterAddons(for: item)
+        return ProductDetailsCellViewModel(item: item, currency: order.currency, product: product, hasAddOns: !addOns.isEmpty)
     }
 }
 
-// MARK: - Localization
+// MARK: - Sections configuration
 //
 private extension ReviewOrderViewModel {
-    enum Localization {
-        static let screenTitle = NSLocalizedString("Review Order", comment: "Title of Review Order screen")
+    /// Product section setup
+    ///
+    var productSection: Section {
+        let rows = order.items.map { Row.orderItem(item: $0) }
```

Comment:
I noticed that if I refund a product in an order, that refunded product still shows up on the Review Order screen. I think that happens because `order.items` includes all items in the original order, including refunded items. We'll have to filter out the refunded items so they aren't included in the product list.